spec/es_spec_helper.rb

```diff
@@ -80,7 +80,7 @@ def self.es_version
   end
 
   def self.es_version_satisfies?(*requirement)
-    es_version = RSpec.configuration.filter[:es_version] || ENV['ES_VERSION'] || ENV['ELASTIC_STACK_VERSION']
+    es_version = nilify(RSpec.configuration.filter[:es_version]) || nilify(ENV['ES_VERSION']) || nilify(ENV['ELASTIC_STACK_VERSION'])
```

Comment:
Note for reviewer: if the single part of the concatenated `||` operator is an empty string then the result is that empty string. So if in the `ENV` there is a `ES_VERSION` but it's empty and the same for the first particle then the `es_version` assume the `""` instead of the version of `ELASTIC_STACK_VERSION`